cfpb/owning-a-home
# Scroll bug fix

Fixes issue where closing step expandable using the 'hide' link at the end of expandable content scrolls user to bottom of page


## Changes

- Scroll to top of expandable when expandable is closed from bottom link

## Testing

- Check out branch
- Run `frontendbuild.sh`
- Navigate to one of the journey pages
- Open a step expandable and then close it from the bottom `hide` link
- Check that you have been scrolled to top of expandable when it's closed

## Review

- @cfarm 

## Checklist
